qcschema/dev/dev_schema.py

```diff
@@ -10,17 +10,16 @@
 
 # The base schema definition
 base_schema = {
-    "$schema": "http://json-schema.org/draft-06/schema#",
-    "name": "QC_JSON",
-    "version": "0.1.dev",
-    "url": "http://schema_host.org/schemas/0/something.schema",
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
why use draft-04, not 06? from my uninformed view, the 06 changes (http://json-schema.org/draft-06/json-schema-release-notes.html) look pretty good.

Reply:
I do not believe we use anything in 6 at the moment, so might as well back down the compatibility requirements until we do.

Reply:
Ok, we'll just be on-guard for any field names involving `id`.